scripts/docker/install_airflow.sh

```diff
@@ -48,18 +48,12 @@ function install_airflow() {
     # Determine the installation_command_flags based on AIRFLOW_INSTALLATION_METHOD method
     local installation_command_flags
     if [[ ${AIRFLOW_INSTALLATION_METHOD} == "." ]]; then
-        # We need _a_ file in there otherwise the editable install doesn't include anything in the .pth file
```

Comment:
This is not needed any more - because we are skipping the whole Dockerfile fragment where we only copied and installed from `pyproject.toml` file.